frontend/.snyk

```diff
@@ -16,10 +16,14 @@ ignore:
         expires: '2019-06-12T23:18:30.879Z'
   SNYK-JS-LODASH-450202:
     - newrelic > async > lodash:
-        reason: 'Upgrade path did not work, patching for now'
-        expires: '2020-01-25T16:18:13.016Z'
+        reason: ignore for 30 days
+        expires: '2019-11-01T18:45:56.815Z'
```

Comment:
You reduced the expiration on this one. Please keep it at what it was so we don't have to fix this again literally tomorrow.